archivers/lzo2/Portfile

```diff
@@ -1,14 +1,17 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
Not sure you meant to include this and the whitespace below in this PR ... guessing they were "collateral changes".

Reply:
Yes it was intentional. If i come across ports without it I tend to just add it.